src/models/connectionInfo.ts

```diff
@@ -50,17 +52,29 @@ export function fixupConnectionCredentials(connCreds: IConnectionInfo): IConnect
 	}
 
 	if (isAzureDatabase(connCreds.server)) {
-		// always encrypt connection if connecting to Azure SQL
-		connCreds.encrypt = true;
+		// always encrypt connection when connecting to Azure SQL
+		connCreds.encrypt = EncryptOptions.Mandatory;
 
-		// Ensure minumum connection timeout if connecting to Azure SQL
+		// Ensure minumum connection timeout when connecting to Azure SQL
 		if (connCreds.connectTimeout < Constants.azureSqlDbConnectionTimeout) {
 			connCreds.connectTimeout = Constants.azureSqlDbConnectionTimeout;
 		}
 	}
 	return connCreds;
 }
 
+export function updateEncrypt(connection: IConnectionInfo): { connection: IConnectionInfo, status: boolean } {
```

Comment:
Consider renaming status to `updatePerformed` to more clearly indicate what happened.